lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1133,7 +1133,7 @@ def install_puppet_agent_dev_repo_on( hosts, opts )
               # new hotness
               path_chunk = "apple/#{version}/#{opts[:puppet_collection]}/#{arch}"
               release_path << path_chunk
-              release_file = "#{mac_pkg_name}-1.#{codename}.dmg"
+              release_file = "#{mac_pkg_name}-1.osx#{version}.dmg"
```

Comment:
This will break acceptance running against `puppet-agent-1.2.4`. Probably best to to check the newest hotness `#{mac_pkg_name}-1.osx#{version}.dmg`, then the new hotness `#{mac_pkg_name}-1.#{codename}.dmg`, then the old hotness `#{mac_pkg_name}-#{variant}-#{version}-x86_64.dmg`